kds/kds.go

```diff
@@ -426,7 +509,7 @@ func VcekCertificateExtensions(cert *x509.Certificate) (*Extensions, error) {
 	if exts.CspID != "" {
 		return nil, fmt.Errorf("unexpected CSP_ID in VCEK certificate: %s", exts.CspID)
 	}
-	if len(exts.HWID) != abi.ChipIDSize {
+	if len(exts.HWID) != int(TCBHwIdLenVersion0) && len(exts.HWID) != int(TCBHwIdLenVersion1) {
```

Comment:
You can save a few CPU instructions here.
```suggestion
        var extsLen int = len(exts.HWID)
	if extsLen != int(TCBHwIdLenVersion0) && extsLen != int(TCBHwIdLenVersion1) {
```